acquire/gui/win32.py

```diff
@@ -0,0 +1,469 @@
+
+    def choose_folder(self) -> None:
+        if self._closed:
+            return
+
+        browseinfo = BROWSEINFOA()
+        browseinfo.hwndOwner = self.hwnd
+        browseinfo.pidlRoot = None
+        browseinfo.pszDisplayName = cast(create_string_buffer(b"", size=1000), w.LPSTR)
+        browseinfo.lpszTitle = cast(create_string_buffer(b"Acquire"), w.LPCSTR)
+        browseinfo.ulFlags = 0
+        browseinfo.lpfn = BFFCALLBACK(_bffcallback)
+        browseinfo.lParam = 0
+        browseinfo.iImage = 0
+        choice = shell32.SHBrowseForFolderA(byref(browseinfo))
+        path = create_string_buffer(b"", size=1000)
+        shell32.SHGetPathFromIDList(choice, path)
+        pathstr = string_at(path).decode("utf-8")
+        if pathstr:
+            self.folder = Path(pathstr)
+            user32.SetWindowTextA(self.label, string_at(path))
+            user32.EnableWindow(self.start_button, True)
+
+        # Caller is responsible for freeing this memory.
+        ole32.CoTaskMemFree(choice)
+
+    def show(self) -> None:
+        if self._closed:
+            return
+
+        wndclass = WNDCLASSW()
+        wndclass.style = CS_HREDRAW | CS_VREDRAW
+        wndclass.lpfnWndProc = WNDPROC(_winmessage)
+        wndclass.cbClsExtra = wndclass.cbWndExtra = 0
+        wndclass.hInstance = kernel32.GetModuleHandleW(None)
+        wndclass.hIcon = user32.LoadIconW(None, IDI_APPLICATION)
+        wndclass.hCursor = user32.LoadCursorW(None, IDC_ARROW)
+        wndclass.hbrBackground = gdi32.GetStockObject(WHITE_BRUSH)
+        wndclass.lpszMenuName = None
+        wndclass.lpszClassName = "AcquireGUI"
+        user32.RegisterClassW(byref(wndclass))
+        hwnd = user32.CreateWindowExW(
+            0,
+            wndclass.lpszClassName,
+            "Acquire",
+            WS_OVERLAPPEDWINDOW,
+            CW_USEDEFAULT,
+            CW_USEDEFAULT,
+            600,
+            400,
+            None,
+            None,
+            wndclass.hInstance,
+            None,
+        )
+
+        user32.ShowWindow(hwnd, SW_SHOWNORMAL)
+        user32.UpdateWindow(hwnd)
+        ole32.CoInitialize(None)
+        control_list = INITCOMMONCONTROLSEX()
+        control_list.dwSize = sizeof(INITCOMMONCONTROLSEX)
+        control_list.dwICC = 0x00000020
+        controls_loaded = comctl32.InitCommonControlsEx(byref(control_list))
+
+        if not controls_loaded:
+            raise GUIError("Unable to load GUI controls")
+
+        hFont = gdi32.CreateFontA(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, b"Segoe UI")
+
+        if self.upload_available:
+            self.checkbox = user32.CreateWindowExW(
+                0, "Button", None, WS_CHILD | WS_VISIBLE | BS_AUTOCHECKBOX, 20, 250, 16, 16, hwnd, 0, 0, 0
+            )
+            self.upload_label = user32.CreateWindowExW(
+                0, "static", "upload", WS_CHILD | WS_VISIBLE | SS_LEFT, 50, 250, 100, 32, hwnd, 0, 0, 0
```

Comment:
```suggestion
                0, "static", "Upload", WS_CHILD | WS_VISIBLE | SS_LEFT, 50, 250, 100, 32, hwnd, 0, 0, 0
```
Same thing as before?